support-frontend/assets/helpers/legalCopy.ts

```diff
@@ -1,37 +1,26 @@
 import type { RegularContributionType } from 'helpers/contributions';
 import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
-import {
-	currencies,
-	detect,
-	fromCountryGroupId,
-} from 'helpers/internationalisation/currency';
+import { currencies, detect } from 'helpers/internationalisation/currency';
 import type { Promotion } from 'helpers/productPrice/promotions';
-import { getLowerProductBenefitThreshold } from 'helpers/supporterPlus/benefitsThreshold';
 import { simpleFormatAmount } from './forms/checkouts';
-import type { ActiveProductKey } from './productCatalog';
 
 export const productLegal = (
 	countryGroupId: CountryGroupId,
 	contributionType: RegularContributionType,
 	divider: string,
-	product: ActiveProductKey,
+	thresholdAmount: number,
```

Comment:
@paul-daniel-dempsey I'm interested in what `thresholdAmount` represents and why it's required for the T&Cs? Is it the product price from the product catalog? 

If so, it feels like the "threshold" terminology is a bit if a hangover from the pre-catalog approach to getting prices (eg. when S+ was just a recurring contribution over a certain amount), so maybe we should rename this?  What do you think?